docs/content/operator/common/archives.mdx

```diff
@@ -66,46 +63,43 @@ state-archive-read-config:
 To enable your node to fallback to an archive in case of lag, add this to your `fullnode.yaml` file:
 
 <Tabs groupId="object-store">
-<TabItem value="s3" label="Amazon S3">
-
-```yaml
-state-archive-read-config:
-  - object-store-config:
-      object-store: "S3"
-      # Use iota-testnet-archive for testnet 
-      # Use iota-mainnet-archive for mainnet
-      bucket: "iota-<testnet|mainnet>-archive"
-      aws-access-key-id: "<AWS-ACCESS-KEY-ID>"
-      aws-secret-access-key: "<AWS-SECRET-ACCESS-KEY>"
-      aws-region: "<AWS-REGION>"
-      object-store-connection-limit: 20
-    # How many objects to read ahead when catching up  
-    concurrency: 5
-    # Whether to prune local state based on latest checkpoint in archive.
-    # This should stay false for most use cases
-    use-for-pruning-watermark: false
-```
-
-</TabItem>
-<TabItem value="gcs" label="Google Cloud Storage">
-
-```yaml
-state-archive-read-config:
-  - object-store-config:
-      object-store: "GCS"
-      # Use iota-mainnet-archive for mainnet
-      # Notice there is no archive bucket setup for testnet in GCS
-      bucket: "iota-<testnet|mainnet>-archive"
-      google-service-account: "</path/to/service/account/credentials>"
-      object-store-connection-limit: 20
-    # How many objects to read ahead when catching up  
-    concurrency: 5
-    # Whether to prune local state based on latest checkpoint in archive.
-    # This should stay false for most use cases
-    use-for-pruning-watermark: false
-```
-
-</TabItem>
+  <TabItem value="s3" label="Amazon S3">
+    ```yaml
+    state-archive-read-config:
+      - object-store-config:
+          object-store: "S3"
+          # Use iota-testnet-archive for testnet 
+          # Use iota-mainnet-archive for mainnet
+          bucket: "iota-<testnet|mainnet>-archive"
```

Comment:
we are not using AWS S3 for public archive usage. users need to use `aws-endpoint` instead and enable virtual-hosted style. you can refer to [this](https://github.com/iotaledger/iota/blob/95a0fe8cd0bee5969afcb480072664b750a15bb1/setups/validator/validator-testnet.yaml#L52)